packages/entity-list/src/modules/searchForm/sagas.js

```diff
@@ -187,3 +194,90 @@ export function* getSearchFormValues() {
     }
   ), {})
 }
+
+export function* saveSearchFilter() {
+  const answerChannel = yield call(channel)
+  yield put(notifier.modalComponent(
+    'filter-save',
+    'client.entity-list.search.settings.saveAsFilter',
+    null,
+    ({close}) => {
+      const onSave = value => {
+        close()
+        answerChannel.put(value)
+      }
+      const [name, setName] = useState('')
+      return <>
+        <StatedValue label={'Name'}>
+          <EditableValue type={'string'} value={name} events={{onChange: setName}}/>
+        </StatedValue>
+        <Button onClick={() => onSave(name)} look={'raised'} disabled={!name}>
```

Comment:
Is there a task for yilmaz? I think this buttons must be primary and right aligned.

Reply:
I've created TOCDEV-3688